addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/thermostat/NATherm1Handler.java

```diff
@@ -159,8 +159,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
-    private Integer getSetpointDefaultDuration() {
-        return (Integer) config.get(SETPOINT_DEFAULT_DURATION);
+    private int getSetPointDefaultDuration() {
+        return ((Number) config.get(SETPOINT_DEFAULT_DURATION)).intValue();
```

Comment:
This gave troubles because (at one user?) it was an BigDecimal instead of an Integer causing the cast problems, this should resolve it.